engine/src/main/java/io/camunda/zeebe/engine/processing/job/JobFailProcessor.java

```diff
@@ -167,7 +167,8 @@ private void raiseIncident(final long key, final JobRecord value) {
         .setElementId(value.getElementIdBuffer())
         .setElementInstanceKey(value.getElementInstanceKey())
         .setJobKey(key)
-        .setVariableScopeKey(value.getElementInstanceKey());
+        .setVariableScopeKey(value.getElementInstanceKey())
+        .setTenantId(value.getTenantId());
```

Comment:
❌ You missed setting the tenant in one place: `JobThrowErrorProcessor.raiseIncident()`

Reply:
This also means we don't have a test case for this. It'd be good to add this

Reply:
:+1: Nice catch!